source/assets/stylesheets/all.css.scss

```diff
@@ -0,0 +1,34 @@
+@charset "UTF-8";
+@import url(https://fonts.googleapis.com/css?family=Raleway:300,400);
```

Comment:
You can drop this and add directly on the main layout with `= stylesheet_link_tag 'https://fonts.googleapis.com/css?family=Raleway:300,400'` #performanceMatters 😄